examples/Tutorial 5 - Graph Coloring Solver.ipynb

```diff
@@ -0,0 +1,207 @@
+      "text/plain": [
+       "<Figure size 640x480 with 1 Axes>"
+      ]
+     },
+     "metadata": {},
+     "output_type": "display_data"
+    }
+   ],
+   "source": [
+    "pos = {i: (loader.coordinates_dataset[i][1], loader.coordinates_dataset[i][0]) for i in range(len(loader.coordinates_dataset))}\n",
+    "nx.draw(instance.graph, pos, with_labels=True, node_size=400, node_color='lightblue', font_size=10, font_color='black', edge_color='gray')\n"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "f209d04d",
+   "metadata": {},
+   "source": [
+    "## Solving the RMP Problem\n",
+    "We will use the greedy heuristic algorithm described in [Appendix A](https://arxiv.org/pdf/2301.02637) to find a coloring of the graph using MIS output at each iteration.\n",
+    "\n",
+    "Notice that we test our algorithm on the first 100 antennas of the dataset."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "14077e7c",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "class GraphColoringSolver:\n",
```

Comment:
This can go in the repo. Maybe under `mis/coloring/coloring.py` ? Could you take that opportunity to add docstrings ?